packages/core/src/components/slider/multiSlider.tsx

```diff
@@ -0,0 +1,446 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import classNames from "classnames";
+import * as React from "react";
+
+import { Classes, Intent } from "../../common";
+import { AbstractPureComponent } from "../../common/abstractPureComponent";
+import { intentClass } from "../../common/classes";
+import * as Errors from "../../common/errors";
+import { IProps } from "../../common/props";
+import * as Utils from "../../common/utils";
+import { Handle } from "./handle";
+import { ISliderHandleProps, SliderHandle, SliderHandleInteractionKind, SliderHandleType } from "./sliderHandle";
+import { SliderTrackStop } from "./sliderTrackStop";
+import { argMin, fillValues, formatPercentage } from "./sliderUtils";
+
+export interface ICoreSliderProps extends IProps {
+    /**
+     * Whether the slider is non-interactive.
+     * @default false
+     */
+    disabled?: boolean;
+
+    /**
+     * Increment between successive labels. Must be greater than zero.
+     * @default 1
+     */
+    labelStepSize?: number;
+
+    /**
+     * Number of decimal places to use when rendering label value. Default value is the number of
+     * decimals used in the `stepSize` prop. This prop has _no effect_ if you supply a custom
+     * `labelRenderer` callback.
+     * @default inferred from stepSize
+     */
+    labelPrecision?: number;
+
+    /**
+     * Maximum value of the slider.
+     * @default 10
+     */
+    max?: number;
+
+    /**
+     * Minimum value of the slider.
+     * @default 0
+     */
+    min?: number;
+
+    /**
+     * Whether a solid bar should be rendered on the track between current and initial values,
+     * or between handles for `RangeSlider`.
+     * @default true
+     */
+    showTrackFill?: boolean;
```

Comment:
@jscheiny is this prop respected by your impl?

Reply:
Yes, here: https://github.com/palantir/blueprint/pull/2536/files#diff-b1bce65425b54a1075cd383becfa1663R258